web/client/components/catalog/RecordItem.jsx

```diff
@@ -62,18 +87,81 @@ const RecordItem = React.createClass({
+        const copyToClipboardTooltip = (<Tooltip id="tooltip"><Message msgId="catalog.copyToClipboard"/></Tooltip>);
+        const copiedTooltip = (<Tooltip id="tooltip"><Message msgId="catalog.copied"/></Tooltip>);
+        let getCapabilitiesLinks = [];
+        // creating get capbilities links that will be used to share layers info
+        if (this.props.showGetCapLinks) {
+            // helper function to create a get capabilities link componeent
+            let createCapabilitiesLink = function(key, getCapabilitiesUrl, messageId, onCopy, tooltip, setLinkCopiedStatus) {
+                return (
+                    <div key={key} className="input-group get-cap-links">
+                        <input className="form-control input-sm cap-input" type="text"
+                               value={getCapabilitiesUrl} onChange={() => {}}/>
+                        <span className="input-group-btn">
+                            <CopyToClipboard text={getCapabilitiesUrl} onCopy={onCopy}>
+                                <OverlayTrigger placement="right" overlay={tooltip} onExited={() => setLinkCopiedStatus(key, false)}>
+                                    <Button bsSize="small" className="cap-button" onClick={() => setLinkCopiedStatus(key, true)}>
+                                        <Glyphicon glyph="copy"/>&nbsp;<Message msgId={messageId}/>
+                                    </Button>
+                                </OverlayTrigger>
+                            </CopyToClipboard>
+                        </span>
+                    </div>
+                );
+            };
+            let wmsGetCap = head(record.references.filter(reference => reference.type &&
+                reference.type.indexOf("OGC:WMS") > -1 && reference.type.indexOf("http-get-capabilities") > -1));
+            let wfsGetCap = head(record.references.filter(reference => reference.type &&
+                reference.type.indexOf("OGC:WFS") > -1 && reference.type.indexOf("http-get-capabilities") > -1));
+            if (wmsGetCap) {
+                let key = "wms-get-cap-link";
+                getCapabilitiesLinks.push(createCapabilitiesLink(key, wmsGetCap.url, "catalog.wmsGetCapLink",
+                    this.props.onCopy, this.isLinkCopied(key) ? copiedTooltip : copyToClipboardTooltip, this.setLinkCopiedStatus));
+            }
+            if (wfsGetCap) {
+                let key = "wfs-get-cap-link";
+                getCapabilitiesLinks.push(createCapabilitiesLink(key, wfsGetCap.url, "catalog.wfsGetCapLink",
+                    this.props.onCopy, this.isLinkCopied(key) ? copiedTooltip : copyToClipboardTooltip, this.setLinkCopiedStatus));
+            }
+            if (getCapabilitiesLinks.length > 0) {
+                buttons.push(
+                    <OverlayTrigger key="copy-caps" container={this} trigger="click" placement="left" overlay={
+                            <Popover id="copy-links-popover">{getCapabilitiesLinks}</Popover>}>
```

Comment:
Also this can be a separate component that gets an array of objects (url, label) and shows the proper inputs, or something like that. It should make easier to test it